search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/llm/DefaultLlmImpl.java

```diff
@@ -119,19 +121,35 @@ protected Map<String, String> getInputParameters(ChatCompletionInput chatComplet
                         )
                 );
         } else {
-            throw new IllegalArgumentException("Unknown/unsupported model provider: " + chatCompletionInput.getModelProvider());
+            throw new IllegalArgumentException(
+                "Unknown/unsupported model provider: "
+                    + chatCompletionInput.getModelProvider()
+                    + ".  You must provide a valid model provider or llm_response_field."
```

Comment:
Not sure if its worth doing but maybe could add a list of the supported providers?

Reply:
I would just document it.  I don't want to update this string every time we add a new provider.